frontend/src/Series/Details/EpisodeRow.js

```diff
@@ -193,6 +195,17 @@ class EpisodeRow extends Component {
               );
             }
 
+            if (name === 'customFormatScore') {
```

Comment:
Thoughts on having a tooltip to show the formats as well? That way you could have 1 column with both the score and formats (on mouse over) without having to have formats as it's own separate column

Reply:
Thats a great idea! Thanks for the snippet below